chore: Update setup-app.sh for Communities

calebpanza opened the pull request:
## Motivation
The Community Features are run off of Stream Activity Feeds which require a few new `env` variables be added. They were added locally and worked out just fine, so this PR exposes them as a part of the main app so we can test in Prod

redreceipt commented on scripts/setup-app.sh:
```diff
@@ -25,7 +25,11 @@ AMPLITUDE_KEY=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text
 
-if [ -z "$APP" ] || [ -z "$ICON" ] || [ -z "$UNIVERSAL_LINK_HOST" ] || [ -z "$DEEP_LINK_HOST" ] || [ -z "$WORDMARK" ] || [ -z "$IOS_BUNDLE" ] || [ -z "$ANDROID_BUNDLE" ] || [ -z "$SERVER_URL" ] || [ -z "$GOOGLE_MAPS_KEY" ] || [ -z "$THEME" ] || [ -z "$APPLE_TEAM_ID" ] || [ -z "$WORDMARK_DARK" ]; then
+# Stream Activity Feed
+STREAM_FEED_API_KEY=$(curl -s --fail -H "x-api-key: $STREAM.API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.SENTRY_TOKEN")
+STREAM_FEED_APP_ID=$(curl -s --fail -H "x-api-key: $STREAM.STREAM.APP_ID" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.SENTRY_TOKEN")
```
```suggestion
STREAM_FEED_API_KEY=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/STREAM.API_KEY")
STREAM_FEED_APP_ID=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/STREAM.STREAM.APP_ID")
```

calebpanza commented:
@redreceipt thanks for clarifying that! I also updated the keys cause I updated the keys in the dB to be a bit more explicit (stream also has a Chat product that we'll likely implement at some point)
